homepairs/homepairsUI/src/state/header/reducer.tsx

```diff
@@ -42,7 +43,7 @@ export const header = (
     action: HeaderAction, 
 ) => {
     /* * NOTE: USE IMMUTABLE UPDATE FUNCTIONS FOR REDUCERS OR ELSE REDUX WILL NOT UPDATE!!! * */
-    const newState = {...state, modalOpen: true };
```

Comment:
same as before, consult Cesar if this is a good change

Reply:
This is fine. Remember the reducer state does not affect any of the modals used. We have given components the power of taking care of that. If one needs to close or open, our actions will invoke a components callback to deal with this.